hive/src/main/java/com/netease/arctic/hive/catalog/MixedHiveTables.java

```diff
@@ -247,6 +247,10 @@ public void dropTableByMeta(TableMeta tableMeta, boolean purge) {
     }
   }
 
+  public void dropTableMetaButNotDropHiveTable(TableMeta tableMeta) {
```

Comment:
Does this method appear to be necessary?
You can just call `MixedTables#dropTableByMeta` directly.